wfl/generate/md/__init__.py

```diff
@@ -78,6 +81,8 @@ def _sample_autopara_wrappable(atoms, calculator, steps, dt, temperature=None, t
     -------
         list(Atoms) trajectories
     """
+	
+    assert integrator.lower() in ["default", "langevin"] 
```

Comment:
Please drop the `.lower()` calls and just make integrator be case sensitive?  Everything else in python is.

Also, I think it's probably worth using the full ASE class names, i.e. `Langevin` and `NVTBerendsen`?